package.json

```diff
@@ -12,7 +12,7 @@
     "prettier": "1.16.4",
     "remap-istanbul": "^0.9.5",
     "shx": "0.2.2",
-    "snyk": "^1.82.1",
+    "snyk": "1.358.0",
```

Comment:
Wonder why CI hasn't caught it

Reply:
Probably 2 things depending on the job - circleci dependency cache, and installing non dev dependencies in the build-all phase.